src/aggregated_non_cooperative.jl

```diff
@@ -221,8 +223,8 @@ function finalize_results!(::AbstractGroupANC, ECModel::AbstractEC)
     # Total reward awarded to the community at each time step
     ECModel.results[:R_Reward_agg] = JuMP.Containers.DenseAxisArray(
         [
-            profile(market_data, "energy_weight")[t] * profile(market_data, "time_res")[t] *
-                profile(market_data, "reward_price")[t] * ECModel.results[:P_shared_agg][t]
+            profile(ECModel.gen_data,"energy_weight")[t] * profile(ECModel.gen_data, "time_res")[t] *
+                market_profile_by_user(ECModel,u_standard, "reward_price")[t] * ECModel.results[:P_shared_agg][t]
```

Comment:
In the gen_data, we may add a category specifying the market_type of the community that specifies the reward. This could be handy to easily test different reward prices for example

Reply:
Updated in the new version